ntoskrnl/ke/bug.c

```diff
@@ -626,7 +626,7 @@ KiDisplayBlueScreen(IN ULONG MessageId,
         InbvResetDisplay();
 
         /* Display blue screen */
-        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_BLUE);
+        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_GREEN);
```

Comment:
Unrelated, please revert.

Reply:
Updated. Please Review.

Reply:
I don't see it updated.

Reply:
Probably updated the wrong branch.
:frownface:

Reply:
```suggestion
        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_BLUE);
```
Feel free to accept this change with single-click in GitHub ;)